ovs/lib/generic.py

```diff
@@ -391,6 +433,14 @@ def _verify_mds_config(current_vdisk):
                                 GenericController._logger.warning('Scrubber - vPool {0} - StorageRouter {1} - vDisk {2} - Skipping because master MDS still not local'.format(vpool.name, storagerouter.name, vdisk.name))
                                 continue
 
+                        # Check if vDisk is already being scrubbed
+                        with volatile_mutex('execute_scrub_work', wait=5):
+                            volatile_key = 'scrub_vdisk_{0}'.format(vdisk_guid)
```

Comment:
Let's add `ovs_` to the key, just in case the memcache is used for some other purpose as well.